lib/lob/resources/resource_base.rb

```diff
@@ -92,7 +92,7 @@ def build_nested_query(value, prefix = nil)
       end
 
       def base_url
-        "#{config[:protocol]}://#{config[:api_key]}:@#{config[:api_host]}/v1"
```

Comment:
I don't know if there was a reason, but from what I've seen none of the other wrappers allow their users to set these options, so I removed it in this wrapper as well. And I can't think of a case where users would need to specify different protocol and hosts, but maybe @mgartner does?